core/state_transition.go

```diff
@@ -385,11 +386,11 @@ func (st *StateTransition) TransitionDb() (ret []byte, usedGas uint64, quotaUsed
 	//normal gas
 	st.state.AddBalance(st.cvm.Coinbase, new(big.Int).Mul(new(big.Int).SetUint64(gu), st.gasPrice))
 
-	quota := big.NewInt(0) //default used 4 k quota every tx for testing
+	quota := uint64(0) //default used 4 k quota every tx for testing
 	if vmerr == nil && st.uploading() {
-		quota = Min(new(big.Int).SetUint64(params.PER_UPLOAD_BYTES), st.state.Upload(st.to()))
+		quota = math2.Uint64Min(params.PER_UPLOAD_BYTES, st.state.Upload(st.to()).Uint64())
 
-		st.state.SubUpload(st.to(), quota) //64 ~ 1024 bytes
+		st.state.SubUpload(st.to(), big.NewInt(int64(quota))) //64 ~ 1024 bytes
```

Comment:
```big.NewInt(0).SetUint64(value)```
The ```upload``` also can be changed to uint64 for it should always below the quota

Reply:
got it

Reply:
got it